lib/Tooltip/Tooltip.js

```diff
@@ -156,7 +156,16 @@ module.exports = kind(
 		* @default null
 		* @public
 		*/
-		contentUpperCase: null
+		contentUpperCase: null,
+
+		/**
+		* The property is introduced to support multiline tooltip
+		* when state is declared, it will create a multiline popup.  
+		* @type {string}
+		* @default undefined
+		* @public
+		*/
+		state: undefined
```

Comment:
@otisnate Yes. It should be `null` . I will change it. 
@dmikey  It can be achieved, but tool-tip design is based on one line. So it takes more effort (should know the internals too). The position of the  tip of the tooltip and the height of the tooltip (block div) are highly dependent on each other in the current design. So that's the first hurdle i see for the app developer to use HTML spans. 
'subText'  sounds a suitable name. you are right about `state` doesn't give that feel of content.  But based on the app requirement, when I gave the name `state`, I was not expecting of any random string. The values of the property are some predefined limited no.of strings. 
However, we are looking for more use cases for this scenario, once its done I will change the name to a more appropriate one.
Thankyou.